### DISCUSSION TITLE
add  support reverse_cost to TRSP Restrictions SQL

### DISCUSSION THREAD

**Discussion by gitlsl** [pgRouting/pgrouting#2492] [General]
just like Edges SQL , I suggest add reverse_cost to Restrictions SQL will More convenient 

In addition, . During the path planning process,how follow straight lines and avoid turning corners. Is there any good way to achieve this？
now. I add cost of turning corners  in Restrictions SQL

**Comment by cvvergara** [comment]
Maybe you can give an example, preferably using the sample data of the documentation, how the restrictions sql would look, and examples of running a query giving what would be the expected output.
